app/scripts/setup.sh

```diff
@@ -74,18 +74,24 @@ function createSymlinks() {
     esac
 }
 
-function checkoutSilme() {
-    # Check out SILME library to a specific version (0.8.0)
-    if [ ! -d $libraries/silme/.hg ]
+function setupExternalLibraries() {
+    # Check out or update compare-locales library
+    version="RELEASE_1_2_1"
```

Comment:
OK. Let me know when you've been able to give it a try, then we'll merge it and let it bake a bit on my dev server.